Set max dhcp reply size to something we can pass to NBPs

diff -r 56139ecce1bb tools/firmware/etherboot/ipxe-v1.0.0/src/net/udp/dhcp.c
--- a/tools/firmware/etherboot/ipxe-v1.0.0/src/net/udp/dhcp.c	Tue Feb 09 17:17:35 2010 +0000
+++ b/tools/firmware/etherboot/ipxe-v1.0.0/src/net/udp/dhcp.c	Tue Feb 09 17:18:48 2010 +0000
@@ -73,7 +73,7 @@
 static uint8_t dhcp_request_options_data[] = {
 	DHCP_MESSAGE_TYPE, DHCP_BYTE ( 0 ),
 	DHCP_MAX_MESSAGE_SIZE,
-	DHCP_WORD ( ETH_MAX_MTU - 20 /* IP header */ - 8 /* UDP header */ ),
+	DHCP_WORD ( 1260 ),
 	DHCP_CLIENT_ARCHITECTURE, DHCP_WORD ( 0 ),
 	DHCP_CLIENT_NDI, DHCP_OPTION ( 1 /* UNDI */ , 2, 1 /* v2.1 */ ),
 	DHCP_VENDOR_CLASS_ID,
